Automattic/jetpack
# stats: Fix time period selection

#### Changes proposed in this Pull Request:

PR #19310 changed an `in_array()` call to strict comparison, which broke
because the call was comparing the string input with integer values in
the array.

Fixes #19455

#### Jetpack product discussion


p1617890154052100-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Go to wp-admin, Jetpack -> Site Stats. Click the "Weeks" and "Months" tabs, the graph should change.
